using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -1,150 +1,271 @@
-# Adobe Campaign Standard API reference
+# Campaign API reference
 
-## Version of the Campaign Standard extension
+---
 
-The `extensionVersion()` API returns the version of the Campaign Standard extension that is registered with the Mobile Core extension.
+### extensionVersion
 
-To get the version of the Campaign Standard extension, use the following code sample:
+Returns the running version of the AEPCampaign extension.
```

Comment:
Can we keep this independent of the extension implementation `... of the Campaign extension.` ?